--- SDL2-2.0.10/src/audio/coreaudio/SDL_coreaudio.m	2019-07-24 22:32:36.000000000 -0600
+++ SDL2-2.0.10-goob/src/audio/coreaudio/SDL_coreaudio.m	2019-12-07 23:07:03.000000000 -0700
@@ -680,17 +680,17 @@
 
     /* Make sure we can feed the device a minimum amount of time */
     double MINIMUM_AUDIO_BUFFER_TIME_MS = 15.0;
-#if defined(__IPHONEOS__)
-    if (floor(NSFoundationVersionNumber) <= NSFoundationVersionNumber_iOS_7_1) {
+//#if defined(__IPHONEOS__)
+    //if (floor(NSFoundationVersionNumber) <= NSFoundationVersionNumber_iOS_7_1) {
         /* Older iOS hardware, use 40 ms as a minimum time */
-        MINIMUM_AUDIO_BUFFER_TIME_MS = 40.0;
-    }
-#endif
+        //MINIMUM_AUDIO_BUFFER_TIME_MS = 40.0;
+    //}
+//#endif
     const double msecs = (this->spec.samples / ((double) this->spec.freq)) * 1000.0;
-    int numAudioBuffers = 2;
-    if (msecs < MINIMUM_AUDIO_BUFFER_TIME_MS) {  /* use more buffers if we have a VERY small sample set. */
-        numAudioBuffers = ((int)SDL_ceil(MINIMUM_AUDIO_BUFFER_TIME_MS / msecs) * 2);
-    }
+    int numAudioBuffers = 4;
+    //if (msecs < MINIMUM_AUDIO_BUFFER_TIME_MS) {  /* use more buffers if we have a VERY small sample set. */
+        //numAudioBuffers = ((int)SDL_ceil(MINIMUM_AUDIO_BUFFER_TIME_MS / msecs) * 2);
+    //}
 
     this->hidden->audioBuffer = SDL_calloc(1, sizeof (AudioQueueBufferRef) * numAudioBuffers);
     if (this->hidden->audioBuffer == NULL) {
